src/widgets/mentors-benefit/ui/mentors-benefit.tsx

```diff
@@ -0,0 +1,35 @@
+import classNames from 'classnames/bind';
+import { WidgetTitle } from '@/shared/ui/widget-title';
+import { mentorsBenefitData } from 'data';
+
+import styles from './mentors-benefit.module.scss';
+
+const cx = classNames.bind(styles);
+
+type MentorsBenefitsProps = {
+  benefits: string[];
+  lang?: 'en' | 'ru';
+};
+
+const countSymbols = 60;
```

Comment:
i don't quite understand what `countSymbols` means. Maybe let's name this as a `shortBenefitMaxChars`

Reply:
Done